src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const CountriesPopulation = document.querySelectorAll('.population');
```

Comment:
use `camelCase` for variable names
```suggestion
const countriesPopulation = document.querySelectorAll('.population');
```